docs/source/1.0/spec/core/model.rst

```diff
@@ -1,223 +1,431 @@
-.. _shapes:
+.. _smithy-model:
 
-======
-Shapes
-======
+============
+Smithy model
```

Comment:
I think a better title would help, "The Smithy Semantic Model", "Components of Smithy", or something as it's not clear which "model" you mean without qualification in the title. I think "Smithy overview" to "Smithy concepts" or "components", and I'd change "Semantic model" to "The semantic model"

Reply:
> The title seems very vague, it's not clear which "model" you mean without qualification in the title.

How about "Smithy overview"?

Reply:
I kind of like calling this "The Smithy model" still, and making the first sentence something like:

> The *Smithy model* describes the Smithy semantic model and the files used to create it. Smithy models are used to describe services and data structures.

Reply:
I dislike the overloading, but I can live with it